plugins/techdocs/src/reader/components/Reader.tsx

```diff
@@ -448,6 +456,9 @@ export const useTechDocsReaderDom = (
               bottom: 75px;
               position: fixed;
               width: 16rem;
+              padding-left: 0.4rem;
+              padding-right: 0.4rem;
+              direction: rtl;
```

Comment:
Was it intended for the secondary sidebar to have its scroll on the left as well?